products/arangodb

```diff
@@ -0,0 +1,33 @@
+---
+title: ArangoDB
+category: db
+permalink: /arangodb
+releasePolicyLink: https://arangodb.com/master-services-agreement-inc-may-2023-cs/#eolpolicy
+changelogTemplate: "https://docs.arangodb.com/stable/release-notes/version-__RELEASE_CYCLE__/"
+releaseLabel: "ArangoDB __RELEASE_CYCLE__"
+eolWarnThreshold: 121
+releases:
+-   releaseCycle: "3.10"
```

Comment:
Could you reformat each release cycle by following our [Style Guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide) ?

Reply:
I've tried to do that, but let me know if I missed something

Reply:
LGTM, thx.